TeamCode/src/main/java/org/firstinspires/ftc/teamcode/OpModes/GeneralOpMode.java

```diff
@@ -8,8 +8,8 @@ abstract public class GeneralOpMode extends LinearOpMode {
     public ITDbot robot;
     
     public void initialize() {
-        robot = new ITDbot();
-        robot.Init(hardwareMap,telemetry);
+      // robot = new ITDbot();
```

Comment:
I think we should probably either delete it or keep it. Commenting it out tends to make our code look more messy.

Reply:
Somehow this was changed and is now fixed in the actual merge?? idk yup :) thanks! good catch.